modules/sns/outputs.tf

```
@@ -0,0 +1,3 @@
+output "arn" {

```

Tem pelo menos um outro recurso aqui (subscription) que tem um ARN também.

```suggestion
output "topic_arn" {
  description = "ARN of the SNS topic provisioned by this module."
```